### ISSUE TITLE
Uniform colors are not resolved properly?

### ISSUE THREAD

**Issue by gael-honorez-para** [trianglesplatting/triangle-splatting#31]
Hi,

I have some training data using a uniform green screen, and a uniform white floor.

<img width="843" height="971" alt="Image" src="https://github.com/user-attachments/assets/67a313d1-f56d-4c58-b844-ce9548b3d170" />

Once trained (30000 iterations), the guy is resolved very well, but all the uniform surfaces (floor, green screen, and even the pants a bit) are messed up. 
I've tried to train with --no_dome but it doesn't make any significant difference.
Also, this does not happens with 2DGS.

<img width="1188" height="1234" alt="Image" src="https://github.com/user-attachments/assets/a4a1eb8e-fae5-4a45-8dc7-e034be8ea333" />

**Comment by heldJan** [comment]
I guess the triangles are too slow to move toward the background to cover it. Since triangles can be either soft or hard and are not always as smooth as Gaussians, there is sometimes less gradient flow, causing the triangles to move insufficiently. This is fine in regions with many triangles but doesn’t work well in areas where no triangles are initially present.

I would try increasing the learning rate for the vertex positions (--lr_triangles_points_init) so they move faster. Let me know if that works!

**Comment by gael-honorez-para** [comment]
Thanks, trying now.

**Comment by gael-honorez-para** [comment]
Tried with a value of 0.0036, it's way worst. 
<img width="594" height="851" alt="Image" src="https://github.com/user-attachments/assets/68f93780-aa86-4a2d-b63c-0a945b66ac41" />

**Comment by heldJan** [comment]
Mh, okay. Maybe try setting --no_dome to True. During initialization, this creates a dome of triangles uniformly around the scene. Maybe that helps. Let me know!

**Comment by heldJan** [comment]
Since this issue seems mostly related to the initialization, I would suggest looking into ways to improve the initial point cloud. Possibly exploring improvements to the dome we already used, there’s a good chance it can be adapted better for your use case :)

**Comment by gael-honorez-para** [comment]
I've taken the

<img width="1016" height="1132" alt="Image" src="https://github.com/user-attachments/assets/344bf487-d35c-4254-8f8e-3ef31b4629ff" />

 2dgs point cloud as an input, doesn't seems a lot better :(
